e2e/regression_settings/settings.valid.spec.js

```diff
@@ -0,0 +1,26 @@
+const { loop } = require('../../src/index');
+
+describe('Closed loop is allowed when', () => {
+    let config = {
+        scenario: 'flat_cgm_trace_with_basal',//also need insulin data too ...
+        settings: loop.settings.default
+    };
+    afterAll(async () => {
+        await loop.RemoveData();
+    });
+    it('lauch loop', async () => {
+        await loop.Launch();
+    });
+    it('apply all settings', async () => {
+        await loop.Configure(config);
+    });
+    it('should not be in closed loop mode', async () => {
+        await loop.screens.home.ExpectLoopNotYetRun();
+    });
+    it('should advance the scenario so we are looping', async () => {
+        await loop.AdvanceScenario(config.scenario, '1');
+    });
+    it('should have no status alert', async () => {
+        await loop.screens.home.ExpectNoLoopStatusAlert();
```

Comment:
[nit] The `ExpectNoLoopStatusAlert` is a bit misleading here. I initially read this as expect a "no loop" status alert. Perhaps something like "ExpectSuccessfulLoop" to indicate that it is a positive outcome?

Reply:
good idea